src/server/util/readingsUtils.js

```diff
@@ -75,6 +75,29 @@ function expectReadingToEqualExpected(res, expected, id = METER_ID) {
     }
 }
 
+/**
+ * Compares readings from api call against the expected readings csv (now with min/max)
+ * @param {request.Response} res the response to the HTTP GET request from Chai
+ * @param {array} expected the returned array from parseExpectedCsv
+ */
+function expectMaxMinToEqualExpected(res, expected, id = METER_ID) {
```

Comment:
I think you need to add this function to the export at the end of the file so the test can access. I believe this is why the test failed in CI on GitHub.

Reply:
I did replace the expectReadingToEqualExpected to that function, however it still doesn't work? Maybe I'm doing the importing wrong, since it says the function is still not being read even though I imported it.

Reply:
Should I reset the units like I did in test B15?

Reply:
Since we are talking soon, why don't we discuss this then.